docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -91,6 +91,8 @@ Example:
 | `prometheus_metrics_retention` | Time duration Prometheus shall retain data for. Must match the regular expression `[0-9]+(ms\|s\|m\|h\|d\|w\|y)` (milliseconds, seconds, minutes, hours, days, weeks and years). | `10d` | false |
 | `prometheus_external_url` | The external URL Prometheus instances will be available under. This is necessary to generate correct URLs. This is necessary if Prometheus is not served from root of a DNS name. | "" | false |
 | `prometheus_node_selector` | Node selector to specify nodes where the Prometheus pods should be deployed. | {} | false |
+| `watch_labeled_service_monitors` | By default prometheus operator watches all the ServiceMonitor objects in the cluster. If set to `true` then only ServiceMonitors with label `release: "prometheus-operator"` will be watched. | `true` | false |
```

Comment:
The description of parameter is not correct as far as I understand. We agreed not to collect all of them by default.

Reply:
good catch. I forgot to change this unlike other things.